Setup your first development environment.md

```diff
@@ -0,0 +1,83 @@
+# Setup your first development environment
+
+To be able to learn programming you will need to have some programs that make your life easier.
+No reasonable developer will want to write code inside of Notepad or Word, or share code throught USB drives.
+In this chapter we will help you setup your development environment.
```

Comment:
Probably more fitting with something like
"In this chapter you will learn how to setup your development environment"

"We" wrote it, but technically the readers are helping/learning themselves